static/js/apps/nl_interface/app_state.ts

```diff
@@ -75,15 +75,58 @@ export interface NLQueryContextType {
 }
 
 interface NLAppConfig {
-  autoRun: boolean;
+  /**
+   * Auto play mode option: Automatically run the query; don't wait for the user to hit enter
+   * Enable with URL param: "a=True"
+   * Default: false
+   */
+  autoPlayAutoRunQuery: boolean;
+
+  /**
+   * Auto play mode option: Wait for the user to hit "enter" in the search box before showing the next query
+   * Enable with URL param: "m=True"
+   * Default: false
+   */
+  autoPlayManuallyShowQuery: boolean;
+
+  /**
```

Comment:
&m and &qd are mutually exclusive? Wonder should &m has higher priority.

Reply:
Agreed about m's precedent - added!